config/suppressions.xml

```diff
@@ -74,7 +74,7 @@
               files="AbstractClassNameCheckTest.java|AbstractTypeAwareCheckTest.java|AbstractJavadocCheckTest.java|AbstractViolationReporterTest.java"/>
 
     <!-- Tone down the checking for test code -->
-    <suppress checks="CyclomaticComplexity" files="[\\/]XDocsPagesTest\.java" lines="298"/>
+    <suppress checks="CyclomaticComplexity" files="[\\/]XDocsPagesTest\.java" lines="299"/>
```

Comment:
@mkordas 
The new check was added to `XDocsPagesTest.XML_FILESET_LIST` for proper validation during the XDoc tests, like the other `Regexp` checks listed there. The suppression is there because one of the methods in the class was never fixed for how complicated it is.